package.json

```diff
@@ -1981,6 +1998,32 @@
                     "enablement": "!isCloud9",
                     "group": "0@3"
                 }
+            ],
+            "aws.codecatalyst.submenu": [
+                {
+                    "command": "aws.codecatalyst.manageConnections",
+                    "group": "0@1"
+                },
+                {
+                    "command": "aws.codecatalyst.signout",
+                    "when": "!isCloud9 && CODECATALYST_CONNECTED",
```

Comment:
setContext doesn't actually work in cloud9, but it looks like all of these cases are `!isCloud9` so it doesn't matter in this case.

Reply:
Better to have less uses of `isCloud9`, will remove. Good callout